src/main/java/duke/command/CommandBuilder.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
```

Comment:
This file is missing JavaDocs, did you forget about it?

Reply:
Ah, there are no public methods here, so I didn't see a need to add JavaDoc comments here.